pkg/server/server.go

```diff
@@ -439,6 +440,13 @@ func (s *Server) HTTPHandler() http.Handler {
 				pluginsHandler.HandlePlugins(w, r)
 			}),
 		))
+
```

Comment:
We need to remove the `if len(s.EnabledConsolePlugins) > 0 {` block and register both handlers all the time, since we also should empty array of plugins.

Reply:
@jhadvig done!